src/command.rs

```diff
@@ -49,6 +49,42 @@ macro_rules! delegate {
     }};
 }
 
+/// If a command is `OverSsh` then it can be executed over an SSH session.
+/// Primarily a way to allow `std::process::Command` to be turned directly into an `openssh::Command`.
+pub trait OverSsh {
+    /// Given a session, return a command that can be executed over that session.
+    fn over_session<'session>(&self, session: &'session Session) -> crate::Command<'session>;
+}
+
+impl OverSsh for std::process::Command {
+    /// Given a session, convert a std::process::Command into an `openssh::Command` 
+    /// that can be executed over that session.
+    /// ```no_run
+    /// use std::process::Command;
+    /// use openssh::{Session, KnownHosts, OverSsh};
+
+    /// let session = Session::connect_mux("me@ssh.example.com", KnownHosts::Strict).await?;
+    /// let ls = 
+    ///     Command::new("ls")
+    ///     .arg("-l")
+    ///     .arg("-a")
+    ///     .arg("-h")
+    ///     .over_session(&session)
+    ///     .output()
+    ///     .await?;
+    /// 
+    /// assert!(String::from_utf8(ls.stdout).unwrap().contains("total"));
+    /// 
+    /// 
+    /// ```
+    fn over_session<'session>(&self, session: &'session Session) -> Command<'session> {
+
+        let mut command = session.raw_command(self.get_program());
+        command.raw_args(self.get_args());
```

Comment:
Same here, I think `Command::args` is more appropriate here.

Reply:
same story as with `Session::command` (i.e. expected `Cow<str>` but provided `Cow<OsStr>`).